app/controllers/APIController.scala

```diff
@@ -3,14 +3,21 @@ package controllers
 import services._
 import io.circe.generic.codec.DerivedAsObjectCodec.deriveCodec
 import io.circe.syntax.EncoderOps
+import io.circe._
+
 import javax.inject._
 import play.api._
 import play.api.mvc._
+import DAO.BakeryDB
+import io.circe
+
+import java.util.UUID
 
 @Singleton
 class APIController @Inject() (
-    val environment: Environment,
-    val controllerComponents: ControllerComponents
+    environment: Environment,
+    val controllerComponents: ControllerComponents,
+    bakeryDB: BakeryDB
 ) extends BaseController {
 
   def statusToJson: String = {
```

Comment:
The reason you are having a weird declaration on the status class here is because it is a class, not a case class. please change this to a case class

Reply:
Change APIController into a case class so I can remove the val declaration?

Reply:
no, the "Status" class